deps/rabbitmq_cli/lib/rabbitmq/cli/plugins/commands/list_command.ex

```diff
@@ -20,9 +20,15 @@ defmodule RabbitMQ.CLI.Plugins.Commands.ListCommand do
   def merge_defaults(args, opts), do: {args, Map.merge(default_opts(), opts)}
 
   def switches(),
-    do: [verbose: :boolean, minimal: :boolean, enabled: :boolean, implicitly_enabled: :boolean]
+    do: [
+      verbose: :boolean,
+      minimal: :boolean,
+      enabled: :boolean,
+      implicitly_enabled: :boolean,
+      no_warn: :boolean
```

Comment:
Can we rename this to `ignore_warnings`?

Reply:
done! thanks was hoping for a better name suggestion 😹 👍